shorebird_code_push/README.md

```diff
@@ -4,6 +4,38 @@
 
 A Dart library that allows Flutter apps to get information about code push updates.
 
+## Usage
+
+```dart
+import 'package:shorebird_code_push/shorebird_code_push.dart';
+
+final shorebirdCodePush = ShorebirdCodePush();
```

Comment:
I wonder if we should drop our name from the classes in the package?

Reply:
No strong opinion here. I'd say we just leave it as is for now – it's easy enough to change later

Reply:
It just requires all clients to update, but that's fine too.  We can manage that when we do so.